model/patch/patch.go

```diff
@@ -590,6 +590,19 @@ func (p *Patch) GetRequester() string {
 	return evergreen.PatchVersionRequester
 }
 
+func GetRequesterFromAlias(alias string) string {
```

Comment:
I get why you wanted to add this, but this seems too specific to be a public function. Is there a clean way to get the requester in ResolvePatchVTs?

Reply:
GetProjectAliasResults calls this, so in that case we really only have the alias to go off of. I'll move this to a private function for ResolvePatchVTs, if that works?

Reply:
Actually a private function with the other alias logic.